core/MemoryAccessInfo.hpp

```diff
@@ -100,6 +121,10 @@ namespace olympia {
         // DCache access status
         CacheState cache_access_state_;
 
+        // Src and destination unit name for the packet
+        UnitName src_;
+        UnitName dest_;
```

Comment:
Thanks for doing that!  One thing (for the future or right now, up to you since it's a bit legacy) it's better, IMHO, to initialize these inline:
```suggestion
        UnitName src_ = UnitName::NO_ACCESS;
        UnitName dest_ = UnitName::NO_ACCESS;
```
Reason (not that've run into this issue before or anything 😛 ): if someone adds a new constructor and doesn't take advantage of construction delegation, they _will_ forget to initialize these variables.  I guarantee it!